packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -363,9 +364,13 @@ export class LspController {
         }
     }
 
+    private isAmznLinux2(): boolean {
+        return os.release().includes('amzn2int') && process.platform === 'linux'
```

Comment:
can you use the existing `isAmazonInternalOs` ? https://github.com/aws/aws-toolkit-vscode/blob/e291f42696e934c50a87bd37e9af04f516e11459/packages/core/src/shared/vscode/env.ts#L107-L108